ciphey/basemods/Decoders/tap_code.py

```diff
@@ -0,0 +1,57 @@
+# by https://github.com/RustyDucky
+
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry
+
+
+@registry.register
+class Tap_code(Decoder[str, str]):
+    def decode(self, input: T) -> Optional[U]:
+        try: #So it doesn't throw an error when its not the right format or just not tap code

```

Comment:
When it comes to code style/formatting, I think @brandonskerritt can help you make some changes to bring the code in-line with the codebase :)

Reply:
All code must be run through Python Black :) 
https://github.com/psf/black
This will fix like 99% of the code formatting issues you have :)